### PR TITLE
Adding arguments input field for docker compose task

### PR DESCRIPTION
**Pull request by rakol-ms**
arguments_docker_compose

ref: #7882 *Add support for additional arguments in Docker Compose build task for Build service images action*

### CODE REVIEW

File: Tasks/DockerComposeV0/task.loc.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 0,
-    "Minor": 155,
+    "Minor": 156,
     "Patch": 1
```
Comment (ds-ms) [inline Tasks/DockerComposeV0/task.loc.json]: NIT: Patch 0

File: Tasks/DockerComposeV0/dockercomposebuild.ts [hunk 2]
```diff
@@ -60,6 +61,7 @@ function addOtherTags(connection: DockerComposeConnection, imageName: string): a
 export function run(connection: DockerComposeConnection): any {
     var command = connection.createComposeCommand();
     command.arg("build");
+    command.line(dockerCommandUtils.getCommandArguments(tl.getInput("arguments", false)));
```
Comment (bansalaseem) [inline Tasks/DockerComposeV0/dockercomposebuild.ts]: I worry about null reference with this code. Can you please make tl.getInput('',..) in 1 line and assign it to variable. Based on non-nullity of it get the command arguments in the next line and based on the non-nullity of it, use command.line(…)

File: Tasks/DockerComposeV0/dockercomposedigests.ts [hunk 3]
```diff
@@ -5,10 +5,12 @@ import * as tl from "azure-pipelines-task-lib/task";
 import * as yaml from "js-yaml";
 import DockerComposeConnection from "./dockercomposeconnection";
 import * as imageUtils from "docker-common-v2/containerimageutils";
+import * as dockerCommandUtils from "docker-common-v2/dockercommandutils";
 
 function dockerPull(connection: DockerComposeConnection, imageName: string, imageDigests: any, serviceName: string) {
     var command = connection.createCommand();
     command.arg("pull");
+    command.line(dockerCommandUtils.getCommandArguments(tl.getInput("arguments", false)));
```
Comment (bansalaseem) [inline Tasks/DockerComposeV0/dockercomposedigests.ts]: Same as above

File: Tasks/DockerComposeV0/task.loc.json (hunk 1 above)
Comment (rakol-ms) [reply Tasks/DockerComposeV0/task.loc.json]: Done.

File: Tasks/DockerComposeV0/dockercomposebuild.ts (hunk 2 above)
Comment (rakol-ms) [reply Tasks/DockerComposeV0/dockercomposebuild.ts]: Done. Checked for null references

File: Tasks/DockerComposeV0/dockercomposedigests.ts (hunk 3 above)
Comment (rakol-ms) [reply Tasks/DockerComposeV0/dockercomposedigests.ts]: Done

**Pull request merged**